### PR TITLE
systemvmtemplate: update Debian ISO URL and checksum

### PR DESCRIPTION
**Pull request by rohityadavcloud**
This fixes packer systemvmtemplate build and uses latest Debian ISO URL and checksum.

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
Debian has released new 11.6.0 with dozens of security fixes; https://www.debian.org/News/2022/20221217
We need to check if we should get this for 4.18 cc @DaanHoogland

**Review by weizhouapache — COMMENTED** [review]
code lgtm

@rohityadavcloud
Do we need to update the systemvm template version in pom.xml ?

**Comment by rohityadavcloud** [comment]
Thanks @weizhouapache fixed pom.xml

File: pom.xml [hunk 1]
```diff
@@ -49,7 +49,7 @@
         <!-- keep in alphabetic order -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <project.systemvm.template.version>4.17.2.0</project.systemvm.template.version>
+        <project.systemvm.template.version>4.18.0.0</project.systemvm.template.version>
```
Comment (rohityadavcloud) [inline pom.xml:52]: I'll need to build the systemvmtemplate and kick the smoketests/pkging. Due to missing systemvmtemplate metadata ini/checksums, build is failing.

**Comment by DaanHoogland** [comment]
looks good @rohityadavcloud.

**Comment by shwstppr** [comment]
@rohityadavcloud @DaanHoogland @weizhouapache we need a new template for https://github.com/apache/cloudstack/pull/6426 as well. Can we get (or wait for) that merged before rebuilding a 4.18.0 template?

**Comment by rohityadavcloud** [comment]
4.18 systemvmtemplate built and being uploaded here: http://download.cloudstack.org/systemvm/4.18/
will shortly kick pkging and smoketests cc @DaanHoogland

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Review by weizhouapache — APPROVED** [review]
code lgtm

File: pom.xml (hunk 1 above)
Comment (weizhouapache) [reply pom.xml:52]: thanks @rohityadavcloud for the fix and building, packaging.

**Comment by rohityadavcloud** [comment]
Build logs and templates are here now:
http://download.cloudstack.org/systemvm/4.18/
http://packages.shapeblue.com/systemvmtemplate/4.18/

**Comment by weizhouapache** [comment]
@blueorangutan test matrix

**Comment by rohityadavcloud** [comment]
Errors not caused by this PR

**Review by shwstppr — APPROVED** [review]

**Pull request merged**